src/Components/Patient/ManagePatients.tsx

```diff
@@ -894,9 +894,9 @@ export const PatientManager = () => {
               onClick={() => advancedFilter.setShow(true)}
             />
             <SortDropdownMenu
-              options={PATIENT_SORT_OPTIONS}
+              options={DISCHARGED_PATIENT_SORT_OPTIONS}
```

Comment:
The intention of the issue was to show discharged patient sort options if the tab selected is discharged. (Which is accessible only for state admins through the `ManagePatients` component).